test/client.test.ts

```diff
@@ -563,6 +563,24 @@ describe('Tests ReactNativeClient', () => {
       expect(client.getIntegrationById('MockRoutingInstrumentation')).toBeTruthy();
     });
   });
+
+  describe('register enabled user interactions tracing as integrations', () => {
```

Comment:
M: Can we add a test that we don't add an integration if it's not installed?

Reply:
Thanks. Will add that before merging.